packages/fx-core/tests/core/localSettings.test.ts

```diff
@@ -187,6 +204,16 @@ describe("LocalSettings provider APIs", () => {
       chai.assert.isTrue(localSettings!.teamsApp?.has(key));
     }
 
+    // Verify Simple Auth related settings
+    if (hasSimpleAuth) {
```

Comment:
we may also need to verify when simple auth is disabled, there's no simple auth related configs.